src/hyp3_isce2/insar_tops_burst.py

```diff
@@ -172,7 +192,7 @@ def make_parameter_file(
     slant_range_time = float(ref_annotation_xml.find('.//slantRangeTime').text)
     range_sampling_rate = float(ref_annotation_xml.find('.//rangeSamplingRate').text)
     number_samples = int(ref_annotation_xml.find('.//swathTiming/samplesPerBurst').text)
-    baseline_perp = topsProc_xml.find(f'.//IW-{swath_number}_Bperp_at_midrange_for_first_common_burst').text
+    baseline_perp = topsProc_xml.find(f'.//IW-{reference_scene[12]}_Bperp_at_midrange_for_first_common_burst').text
```

Comment:
Let's make a helper function (e.g. `get_swath_number(reference_scene: str) -> int`) and call it everywhere that `reference_scene[12]` is used (I've seen two places so far).